DSharpPlus/Entities/Interaction/Components/DiscordSectionComponent.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace DSharpPlus.Entities;
+
+public class DiscordSectionComponent : DiscordComponent
+{
+    public new DiscordComponentType Type => DiscordComponentType.Section;
+
+    /// <summary>
+    /// Gets the accessory for this section.
+    ///
+    /// Accessories take the place of a thumbnail (that is, are positioned as a thumbnail would be) regardless of component.
+    /// At this time, only <see cref="DiscordButtonComponent"/> and <see cref="DiscordThumbnailComponent"/> are supported.
+    /// </summary>
```

Comment:
empty lines are ignored in xmldocs, afaik? clarify with either `<br/>` (unlike in html, you cannot omit the closing slash) or with a remarks section